docs/conf.py

```diff
@@ -39,10 +39,15 @@
     'nbsphinx',
     'sphinx.ext.extlinks',
     'sphinx.ext.mathjax',
+    'sphinx_copybutton',
 ]
 ipython_mplbackend = ""
 imgmath_image_format = 'svg'
 
+copybutton_selector = 'div:not(.no-copy)>div.highlight pre'
```

Comment:
```suggestion
copybutton_selector = 'div.copy-button>div.highlight pre'
```

We could make the selector inclusive so that, to get the copy button to show, you have to use:

```rst
.. code-block: console
   :class: copy-button

   $ echo "hi"
```

its maybe an extra hassle, but then its guaranteed to not mess with the existing tutorials